components/ILIAS/Forum/classes/class.ilForum.php

```diff
@@ -1484,8 +1488,9 @@ public function prepareText(string $text, int $edit = 0, string $quote_user = ''
 
         if ($type !== 'export') {
             if ($edit === 0) {
-                $text = ilMathJax::getInstance()->insertLatexImages($text, "\<span class\=\"latex\">", "\<\/span>");
-                $text = ilMathJax::getInstance()->insertLatexImages($text, "\[tex\]", "\[\/tex\]");
+                // todo: uncomment when PR 9870 is merged
+                // $text = ilRTE::replaceLatexSpan($text);
```

Comment:
Hi @fneumann , you could now rebase the branch(es), the code of PR #9870 is now part of the trunk. Afterwards, I will merge the forum-related PR here.